src/firmware/boards/robot_stm32h7/freertos.c

```diff
@@ -140,11 +235,12 @@ const osMessageQueueAttr_t RobotLogProtoQ_attributes = {.name = "RobotLogProtoQ"
 /* USER CODE END FunctionPrototypes */
 
 void io_proto_multicast_startNetworkingTask(void *argument);
-extern void io_proto_multicast_sender_task(void *argument);
-extern void io_proto_multicast_listener_task(void *argument);
-void test_msg_update(void *argument);
+extern void io_proto_multicast_listenerTask(void *argument);
+extern void io_robot_status_task(void *argument);
+extern void io_proto_multicast_senderTask(void *argument);
```

Comment:
inconsistent camelCase vs snake_case

Reply:
this is our firmware convention

`(io|app)_module_name_functionName`

https://github.com/UBC-Thunderbots/Software/blob/master/docs/firmware-architecture-and-design.md

Reply:
I see but what about io_robot_status_task->io_robot_status_senderTask?

Reply:
the robot status task notifies the io_robot_status_senderTask to actually the send the proto

the robot status task is only responsible for collecting robot status

Reply:
collectorTask? Ah well it's fine